agent/agent.go

```diff
@@ -233,34 +237,35 @@ func _main() int {
 	// Start sending events to the backend
 	go eventhandler.HandleEngineEvents(taskEngine, client, stateManager)
 
-	deregisterInstanceStream := event.NewACSDeregisterInstanceStream()
-	deregisterInstanceStream.StartListening(ctx)
+	deregisterInstanceEventStream := eventstream.NewEventStream("Deregister_Instance", ctx)
```

Comment:
Can we follow the CamlCasing convention here too? `DeregisterContainerInstance`?